### PR TITLE
[16.0][MIG] website_forum_subscription: Migration to version 16.0

### PR DESCRIPTION
**Pull request by carolinafernandez-tecnativa**
- Standard migration to v16

@Tecnativa

TT45242

@pedrobaeza @chienandalu

### CODE REVIEW

File: website_forum_subscription/templates/website_forum_templates.xml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+                    <i class="fa fa-bell" /> Subscribe to forum notifications
+                </a>
+            </div>
+            <div
+                t-if="icons_design and not user_public"
+                class="js_follow_icons_container"
+            >
+                <button class="btn text-reset js_unfollow_btn">
+                    <div class="d-flex align-items-center">
+                        <small>Unsubscribe</small><i class="fa fa-fw ml-1" />
```
Comment (chienandalu) [inline website_forum_subscription/templates/website_forum_templates.xml]:
```suggestion
                        <small>Unsubscribe</small><i class="fa fa-fw ms-1" />
```

File: website_forum_subscription/templates/website_forum_templates.xml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+                    </div>
+                </button>
+                <button class="btn text-reset js_follow_btn">
+                    <div class="d-flex align-items-center">
+                        <small>Subscribe</small><i class="fa fa-fw ml-1" />
```
Comment (chienandalu) [inline website_forum_subscription/templates/website_forum_templates.xml]:
```suggestion
                        <small>Subscribe</small><i class="fa fa-fw ms-1" />
```

**Review by chienandalu — CHANGES_REQUESTED** [review]
Take care of bs4 -> bs5 classes changes (https://getbootstrap.com/docs/5.0/migration/#utilities)

A good custome is to use the tooling provided here to automate those checks: https://github.com/OCA/openupgradelib/pull/338

**Comment by carolinafernandez-tecnativa** [comment]
> Take care of bs4 -> bs5 classes changes (https://getbootstrap.com/docs/5.0/migration/#utilities)
> 
> A good custome is to use the tooling provided here to automate those checks: [OCA/openupgradelib#338](https://github.com/OCA/openupgradelib/pull/338)

Hi thanks! How can i use that tool?

**Comment by chienandalu** [comment]
> Hi thanks! How can i use that tool?

You'll have to build a version that includes that patch and then simply:
```python
from openupgradelib.openupgrade_160 import convert_string_bootstrap_4to5

xml = """<odoo><!-- Your xml --></odoo>"""
print(convert_string_bootstrap_4to5(xml))  # You can copy your output from the console directly
```

**Comment by carolinafernandez-tecnativa** [comment]
WIP

**Comment by carolinafernandez-tecnativa** [comment]
> > Hi thanks! How can i use that tool?
> 
> You'll have to build a version that includes that patch and then simply:
> 
> ```python
> from openupgradelib.openupgrade_160 import convert_string_bootstrap_4to5
> 
> xml = """<odoo></odoo>"""
> print(convert_string_bootstrap_4to5(xml))  # You can copy your output from the console directly
> ```



> Take care of bs4 -> bs5 classes changes (https://getbootstrap.com/docs/5.0/migration/#utilities)
> 
> A good custome is to use the tooling provided here to automate those checks: [OCA/openupgradelib#338](https://github.com/OCA/openupgradelib/pull/338)

Goord morning,
Thanks for your help! I have checked and the only change was the one you detected, you can proceed to check it now 
FYI @pedrobaeza

**Review by chienandalu — APPROVED** [review]
:+1:

**Review by pedrobaeza — APPROVED** [review]
Tested and working:

/ocabot merge nobump

**Pull request merged**